lib/gems/pending/util/object_storage/miq_swift_storage.rb

```diff
@@ -0,0 +1,148 @@
+require 'pp'
+# require 'util/miq_object_storage'
+require 'util/mount/miq_generic_mount_session'
+
+class MiqSwiftStorage < MiqObjectStorage
+  attr_reader :container_name
+
+  def self.uri_scheme
+    "swift".freeze
+  end
+
+  def self.new_with_opts(opts)
+    new(opts.slice(:uri, :username, :password))
+  end
+
+  def initialize(settings)
+    super(settings)
+
+    # NOTE: This line to be removed once manageiq-ui-class region change implemented.
+    @bucket_name         = URI(@settings[:uri]).host
```

Comment:
Is this being used in the `FileDepot` code then?  If so, then that is probably why I omitted it by mistake in [my branch](https://github.com/ManageIQ/manageiq-gems-pending/compare/master...NickLaMuro:swift-storage-as-object-storage).

Sorry about that.